pkg/conversions/virtualnode_types_conversion_test.go

```diff
@@ -3790,3 +3790,68 @@ func TestConvert_CRD_VirtualNodeSpec_To_SDK_VirtualNodeSpec(t *testing.T) {
 		})
 	}
 }
+
+func newStringPtr(s string) *string {
+	return &s
+}
```

Comment:
need this to make one liner string ptrs in Go (sucks!)

Reply:
I think we typically use aws.String() for this. https://docs.aws.amazon.com/sdk-for-go/api/aws/

Reply:
ah yes that works, thanks